src/daemon/rpmostreed-transaction-livefs.c

```diff
@@ -659,6 +659,9 @@ livefs_transaction_execute_inner (LiveFsTransaction *self,
         return FALSE;
     }
 
+  /* XXX: right now we don't have a good solution for this */
+  rpmostree_output_message ("WARNING: changes to /etc are not currently preserved!");
```

Comment:
more info? are not preserved in what situation?

Reply:
See the linked issue: https://github.com/projectatomic/rpm-ostree/issues/40. This is not unique to livefs, it just makes it more obvious.

Reply:
i'm just saying that a little more info for the user in this situation wouldn't be bad. i.e. `any changes made to /etc before reboot will not be preserved after reboot!`

Reply:
Gotcha. Tweaked in https://github.com/projectatomic/rpm-ostree/pull/988.